tsconfig.json

```diff
@@ -16,9 +16,10 @@
     "incremental": true,
     "baseUrl": ".",
     "paths": {
-      "@/*": ["./*"]
+      "@/*": ["./*"],
+      "react": [ "./node_modules/@types/react" ]
```

Comment:
Is this needed? see link below...

https://www.typescriptlang.org/tsconfig#types

Reply:
@codingnuclei i dont understand what to do with the link can you help?

Reply:
Typescript by default includes all types in the @types folder in node_modules. So having a @type folder in ts paths should not be needed... we sure it is required here to fix the react loader? I just did a checkout of your pr and removed the tsconfig change and it seemed to work... can you check if it works for you without it?

Reply:
![image](https://github.com/front-doorxyz/front-door-app/assets/56181880/5d568177-e859-47b2-8def-488ad44e98fa)
see it wont let me build without that line